lib/sycamore/sycamore/utils/cache.py

```diff
@@ -16,10 +18,24 @@ def set(self, hash_key: str, hash_value):
         pass
 
     @staticmethod
-    def get_hash_key(data: bytes) -> str:
+    def get_hash_bytes(data: bytes) -> hashlib.sha256:
```

Comment:
Typically, this is a called a "hash context" and not "bytes".  The reason is that it's much bigger than the final hash.  It also has the ability to be updated incrementally, and can be copied for cases where common prefixes are involved.

Bytes from digest() might be  0x30, 0x31, 0x32, 0x33, 0x34
String from hexdigest() might be "30313234" (using 2 characters per byte)
The SHA-256 context is around 104 bytes